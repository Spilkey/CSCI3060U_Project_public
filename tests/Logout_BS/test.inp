loginuser03logout
exit