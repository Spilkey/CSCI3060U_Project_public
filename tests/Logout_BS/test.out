00 user03          BS 000000000